classes/class.ilOpencastPageComponentPluginGUI.php

```diff
@@ -124,32 +124,81 @@ public function __construct()
         parent::__construct();
     }
 
+    private function ensureContext(string $command): void
+    {
+        $this->ensureParameters();
+        // we connot rely on ilCtrl->getCmdClass() here, because it is seems to be "overwritten" by the pageeditor which is really bad practice.
+        $current_comand_class = strtolower(
```

Comment:
typo in variable name `$current_comand_class` should be `$current_command_class`